apps/vast/README.md

```diff
@@ -46,6 +46,19 @@ Start the application with a config file:
 ./pyvast_threatbus.py -c config.yaml
 ```
 
+### Docker
+
+You can also run this app via Docker.
+
+- Build it:
+  ```
+  docker build . -t tenzir/pyvast-threatbus:latest
+  ```
+- Run it (and mount a custom config file into the container):
+  ```
+  docker run --net=host -v /path/to/your/conf.yaml:/opt/tenzir/threatbus/pyvast-threatbus/config.yaml
```

Comment:
```suggestion
  docker run --net=host -v /path/to/your/conf.yaml:/opt/tenzir/threatbus/pyvast-threatbus/ tenzir/pyvast-threatbus
```

Otherwise this doesn't work for me: You need to specify a directory to copy the file into and you must specify a container image to run.

Reply:
As discussed via call and Slack, this is not necessary.